.travis.yml

```diff
@@ -69,6 +69,8 @@ before_script:
 script:
   - phpdbg -qrr vendor/bin/phpunit --coverage-text --coverage-clover build/logs/clover.xml
   - PHP_CS_FIXER_IGNORE_ENV=1 php vendor/bin/php-cs-fixer --diff --dry-run -v fix
+  - chmod +x benchmarks/bin/php_no_xdebug
```

Comment:
Just commit the file go Git with `chmod +x` and it will be fine. No need to run this each time.

Reply:
Thank you. Never used this feature :) Fixed